Data.TRAMS.Tests/Data.TRAMS.Tests.csproj

```diff
@@ -7,15 +7,21 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-        <PackageReference Include="Moq" Version="4.15.2" />
-        <PackageReference Include="xunit" Version="2.4.0" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-        <PackageReference Include="coverlet.collector" Version="1.2.0" />
+        <ProjectReference Include="..\Data.TRAMS\Data.TRAMS.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\Data.TRAMS\Data.TRAMS.csproj" />
+      <Reference Include="Microsoft.Extensions.Caching.Memory, Version=3.1.22.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
+        <HintPath>..\..\..\Program Files\dotnet\shared\Microsoft.AspNetCore.App\3.1.22\Microsoft.Extensions.Caching.Memory.dll</HintPath>
+      </Reference>
+    </ItemGroup>
```

Comment:
Maybe this ItemGroup could be removed now with the ref below (tests still appear to run without it)?

Reply:
now removed